brizo/routes.py

```diff
@@ -274,19 +255,41 @@ def compute():
-                                               resource_group_name=get_env_property('AZURE_RESOURCE_GROUP', 'azure.resource_group'),
-                                               account_name=get_env_property('AZURE_ACCOUNT_NAME', 'azure.account.name'),
-                                               account_key=get_env_property('AZURE_ACCOUNT_KEY', 'azure.account.key'),
-                                               share_name_input=get_env_property('AZURE_SHARE_INPUT', 'azure.share.input'),
-                                               share_name_output=get_env_property('AZURE_SHARE_OUTPUT', 'azure.share.output'),
-                                               location=get_env_property('AZURE_LOCATION', 'azure.location'),
+                                               resource_group_name=get_env_property(
+                                                   'AZURE_RESOURCE_GROUP', 'azure.resource_group'),
+                                               account_name=get_env_property('AZURE_ACCOUNT_NAME',
+                                                                             'azure.account.name'),
+                                               account_key=get_env_property('AZURE_ACCOUNT_KEY',
+                                                                            'azure.account.key'),
+                                               share_name_input=get_env_property(
+                                                   'AZURE_SHARE_INPUT', 'azure.share.input'),
+                                               share_name_output=get_env_property(
+                                                   'AZURE_SHARE_OUTPUT', 'azure.share.output'),
+                                               location=get_env_property('AZURE_LOCATION',
+                                                                         'azure.location'),
                                                # input_mount_point=data.get('input_mount_point'),
                                                # output_mount_point=data.get('output_mount_point'),
                                                docker_image=data.get('docker_image'),
                                                memory=data.get('memory'),
                                                cpu=data.get('cpu')), 200
 
 
+def check_required_attributes(required_attributes, data, method):
+    assert isinstance(data, dict), 'invalid payload format.'
+    logger.info('got %s request: %s' % (method, data))
+    if not data:
+        logger.error('%s request failed: data is empty.' % method)
+        return 'payload seems empty.', 400
+
+    assert isinstance(data, dict), 'invalid `body` type, should be formatted into a dict.'
+
+    for attr in required_attributes:
+        if attr not in data:
+            logger.error('%s request failed: required attr %s missing.' % (method, attr))
+            return '"%s" is required for registering an asset.' % attr, 400
+    return True
```

Comment:
Should either return a single value (bool) or tuple (message, status).
We can omit the status, let the caller decide the http status code.
I suggest returning a message, empty message means everything is fine.
Or, raise exception when something is missing, otherwise, nothing. This way it can be called in the main try-except block and no if statements are necessary.